Ether.Core/Models/VSTS/WorkItemUpdate.cs

```diff
@@ -53,5 +56,21 @@ public bool IsValueCleared()
 
             public bool IsEmpty => string.IsNullOrEmpty(OldValue) && string.IsNullOrEmpty(NewValue);
         }
+
+        public class RelationsContainer
+        {
+            public List<RelationItem> Added { get; set; }
+            public List<RelationItem> Removed { get; set; }
+        }
+
+        public class RelationItem
+        {
+            public string Rel { get; set; }
+            public string Url { get; set; }
+            public string Name => Attributes.ContainsKey("name") ? Attributes["name"] : null;
```

Comment:
Can `Attributes` be `null`? Also, `string.Empty` might be a better choice than `null` 
```suggestion
            public string Name => Attributes != null && Attributes.ContainsKey("name") ? Attributes["name"] : string.Empty;
```